kanidmd_web_ui/pkg/wasmloader.js

```diff
@@ -0,0 +1,7 @@
+// loads the module which loads the WASM. It's loaders all the way down.
+import init, { run_app } from '/pkg/kanidmd_web_ui.js';
```

Comment:
Makes me wonder if we can do a csp hash on this .... 🤔

Reply:
on `kanidmd_web_ui.js` ? Sure, but (Edge, at least) didn't seem to complain.

Reply:
Yeah, as well as the other bits, but I dunno how easy that is. Maybe that's a second PR .... can you add integrity headers inside js like that?

Reply:
Yeah, I made the entire thing pretty flexible so it's doable - I'm just doing a super-lazy string replace on the body at the moment, could do it with a similar thing for that one. Given that it's in the `pkg` dir though - it might be worth looking at some kind of compile-phase thing that updates the file so we don't need *another* middleware handler. 

There's also the issue that I'm likely to run into where IIRC I've got JS minification on in CloudFlare and it's likely to break the hashes as it is...

Reply:
Yeah, no stress mate, thanks for making the project that little bit better :)